package.json

```diff
@@ -3,9 +3,16 @@
     "version": "1.0.0",
     "description": "",
     "license": "BSD-3-Clause",
-    "private": true,
     "main": "build/cjs/lib.js",
     "module": "build/es/lib.js",
+    "private": false,
+    "publishConfig": {
+        "access": "public"
+    },
+    "sideEffects": [
+        "./src/index.js"
```

Comment:
Is this necessary?

Reply:
Do you mean the sideEffects? or the `access: "public"`? (or both).

Reply:
sideEffects - are there actually non-tree-shakable sideEffects in that file...?

Reply:
@amcgee I thought that initializing the config is a side effect?

Reply:
Right.